### PR TITLE
Update 'http' URLs to 'https' wherever possible.

### PR DESCRIPTION
**Pull request by rossigee**
Resolves issue where RPM key cannot be retrieved in secure environments.

https://github.com/DataDog/ansible-datadog/issues/368

### CODE REVIEW

File: tasks/pkg-debian.yml [hunk 1]
```diff
@@ -56,9 +56,6 @@
     state: "absent"
     update_cache: yes
   with_items:
-    - "deb http://apt.datadoghq.com/ stable main"
```
Comment (bkabrda) [inline tasks/pkg-debian.yml]: These three should actually stay, as this task is supposed to ensure that non-https source is *absent*, not present.

**Review by bkabrda — CHANGES_REQUESTED** [review]
Hi 👋 thanks for submitting the PR. This is a good catch, I've missed this while implementing the feature. I requested one change inline, so once that's fixed I can merge.

File: tasks/pkg-debian.yml (hunk 1 above)
Comment (rossigee) [reply tasks/pkg-debian.yml]: :facepalm: i should have spotted that. Patch updated.

**Review by bkabrda — APPROVED** [review]
Looks great now, I'm merging. Thanks for the PR!

**Pull request merged**